mlflow/utils/file_utils.py

```diff
@@ -31,6 +32,18 @@
 except ImportError:
     from yaml import SafeLoader as YamlSafeLoader, SafeDumper as YamlSafeDumper
 
+_logger = logging.getLogger(__name__)
+_TQDM_AVAILABLE = False
+try:
+    from tqdm.notebook import tqdm
+
+    _TQDM_AVAILABLE = True
+except ImportError:
+    _logger.warning(
+        "module not found: tqdm. To show progress bar for artifacts, "
+        "install with `pip install tqdm`"
+    )
```

Comment:
This message is shown when this module is imported, which might annoy or confuse users.

Reply:
Do we need this to show warning message?

Reply:
That's true... Is there a better way to let customer know that they can use progress bar if they have tqdm installed?

Reply:
Can we include `tqdm` in mlflow's dependencies? `tqdm` is tiny ([77.1 kB](https://pypi.org/project/tqdm/#files)), depends on nothing, and